packages/ui/src/components/ChatMessage/ChatMessage.tsx

```diff
@@ -173,6 +178,26 @@ const ChatMessage = ({
           />
         </View>
 
+        {/** we need to show delivery status even if showAuthor is false
+           previously we were only showing delivery status if showAuthor was true
+           (i.e., on the first of a series of messages)
+        */}
+        {!showAuthor &&
+        !!post.deliveryStatus &&
+        post.deliveryStatus !== 'failed' ? (
+          <View position="absolute" right={12} top={8}>
+            <ChatMessageDeliveryStatus status={post.deliveryStatus} />
+          </View>
+        ) : null}
+
+        {!showAuthor && deliveryFailed ? (
```

Comment:
seems like now we only show it when `showAuthor` is false is that what we want?

Reply:
It still shows in `ChatAuthorRow` when `showAuthor` is true